library/Icingadb/Common/BaseListItem.php

```diff
@@ -157,7 +157,7 @@ protected function setMultiselectFilter(Rule $filter)
 
     protected function setDetailFilter(Rule $filter)
     {
-        $this->addAttributes(['data-icinga-detail-filter' => QueryString::render($filter)]);
+        $this->addAttributes(['data-icinga-detail-filter' => '(' . QueryString::render($filter) . ')']);
```

Comment:
There is `Icinga\Module\Icingadb\Common\DetailActions` for this actually. Any user of this method and the one above should use the list's utility methods instead.

Reply:
Okay, I will add `DetailActions` instead. 
The parentheses were missing anyway or not?


> should use the list's utility methods instead

You mean the `DetailActions` ?

Reply:
Take a look into e.g. `BaseHostListItem` on how this handles this. Then I guess you'll know what I mean?